src/components/CardDropdownButton.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Button } from "components/Button";
+import { EllipsisBtnCopy } from "components/styles/Button";
+import Card from "@leafygreen-ui/card";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+
+interface Props {
+  disabled?: boolean;
+  setIsVisibleCard?: (v: boolean) => void;
+  loading?: boolean;
+  isVisibleCard?: boolean;
+  cardItems: JSX.Element[];
+  dataCyBtn?: string;
+  dataCyCard?: string;
+}
+
+export const CardDropdownButton = ({
```

Comment:
`CardDropdownButton` ? confused by this name. where is there a card involved?

Reply:
The CardDropdownButton is the button with ellipsis and the dropdown together. I called the component CardDropdownButton because it uses Card leafygreen component for the dropdown menu. I can change this name to DropdownButton.
<img width="150" alt="Screen Shot 2020-05-07 at 10 33 05 AM" src="https://user-images.githubusercontent.com/10734386/81307072-3472a980-904e-11ea-97e4-07b11ba5fa83.png">